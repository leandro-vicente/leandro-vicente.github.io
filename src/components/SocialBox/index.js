import React from 'react'
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
import PropTypes from 'prop-types'


const SocialMediaBox = (props) => {
  const { iconsAndLinks } = props
  return (
    <ul className="social-media-list">
      {
        iconsAndLinks.map( (social_network, key) => 
          <li key={key}> 
            <a 
              href={social_network.url} 
              target="_blank" 
              rel="noopener noreferrer"
              aria-label={`${social_network.name} link`}>
              <FontAwesomeIcon icon={['fab', social_network.name]} size='2x' />
            </a> 
          </li>
        )
      }
    </ul>
  )
}

SocialMediaBox.propTypes = {
  iconsAndLinks: PropTypes.string.isRequired
}

export default SocialMediaBox