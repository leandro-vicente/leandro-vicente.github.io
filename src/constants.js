const SOCIAL_ICONS_AND_LINKS = [
  {
    'url': 'https://www.linkedin.com/in/leandrovicentesilva/',
    'name': 'linkedin'
  },
  {
    'glyphicon': 'github',
    'url': 'https://github.com/llevvi/',
    'name': 'github'
  },
  {
    'url': 'http://youtube.com/levvitalk',
    'name': 'youtube'
  },
  {
    'url': 'http://instagram.com/levvitalk',
    'name': 'instagram'
  }
]

const SUBSCRIBE_FORM_URL = 'https://goo.gl/forms/A3zRVgzeH3BMD2FJ3'
  
export { SOCIAL_ICONS_AND_LINKS, SUBSCRIBE_FORM_URL }