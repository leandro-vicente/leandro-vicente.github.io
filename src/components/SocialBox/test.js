import React from 'react'
import SocialBox from './index'
import { SOCIAL_ICONS_AND_LINKS } from '../../constants'
import { shallow } from 'enzyme'

describe('SocialBox', () => {
  it('renders as expected', () => {
    const socialMedia = shallow(<SocialBox iconsAndLinks={SOCIAL_ICONS_AND_LINKS} />)
    expect(socialMedia).toMatchSnapshot()
  })
})