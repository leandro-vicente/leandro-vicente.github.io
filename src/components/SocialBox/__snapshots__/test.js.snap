// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialBox renders as expected 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SocialMediaList
    iconsAndLinks={
      Array [
        Object {
          "name": "linkedin",
          "url": "https://www.linkedin.com/in/leandrovicentesilva/",
        },
        Object {
          "glyphicon": "github",
          "name": "github",
          "url": "https://github.com/llevvi/",
        },
        Object {
          "name": "youtube",
          "url": "http://youtube.com/levvitalk",
        },
        Object {
          "name": "instagram",
          "url": "http://instagram.com/levvitalk",
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <li>
          <a
            aria-label="linkedin link"
            href="https://www.linkedin.com/in/leandrovicentesilva/"
            rel="noopener noreferrer"
            target="_blank"
          >
            <FontAwesomeIcon
              border={false}
              className=""
              fixedWidth={false}
              flip={null}
              icon={
                Array [
                  "fab",
                  "linkedin",
                ]
              }
              inverse={false}
              listItem={false}
              mask={null}
              pull={null}
              pulse={false}
              rotation={null}
              size="2x"
              spin={false}
              symbol={false}
              title=""
              transform={null}
            />
          </a>
        </li>,
        <li>
          <a
            aria-label="github link"
            href="https://github.com/llevvi/"
            rel="noopener noreferrer"
            target="_blank"
          >
            <FontAwesomeIcon
              border={false}
              className=""
              fixedWidth={false}
              flip={null}
              icon={
                Array [
                  "fab",
                  "github",
                ]
              }
              inverse={false}
              listItem={false}
              mask={null}
              pull={null}
              pulse={false}
              rotation={null}
              size="2x"
              spin={false}
              symbol={false}
              title=""
              transform={null}
            />
          </a>
        </li>,
        <li>
          <a
            aria-label="youtube link"
            href="http://youtube.com/levvitalk"
            rel="noopener noreferrer"
            target="_blank"
          >
            <FontAwesomeIcon
              border={false}
              className=""
              fixedWidth={false}
              flip={null}
              icon={
                Array [
                  "fab",
                  "youtube",
                ]
              }
              inverse={false}
              listItem={false}
              mask={null}
              pull={null}
              pulse={false}
              rotation={null}
              size="2x"
              spin={false}
              symbol={false}
              title=""
              transform={null}
            />
          </a>
        </li>,
        <li>
          <a
            aria-label="instagram link"
            href="http://instagram.com/levvitalk"
            rel="noopener noreferrer"
            target="_blank"
          >
            <FontAwesomeIcon
              border={false}
              className=""
              fixedWidth={false}
              flip={null}
              icon={
                Array [
                  "fab",
                  "instagram",
                ]
              }
              inverse={false}
              listItem={false}
              mask={null}
              pull={null}
              pulse={false}
              rotation={null}
              size="2x"
              spin={false}
              symbol={false}
              title=""
              transform={null}
            />
          </a>
        </li>,
      ],
      "className": "social-media-list",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "0",
        "nodeType": "host",
        "props": Object {
          "children": <a
            aria-label="linkedin link"
            href="https://www.linkedin.com/in/leandrovicentesilva/"
            rel="noopener noreferrer"
            target="_blank"
          >
            <FontAwesomeIcon
              border={false}
              className=""
              fixedWidth={false}
              flip={null}
              icon={
                Array [
                  "fab",
                  "linkedin",
                ]
              }
              inverse={false}
              listItem={false}
              mask={null}
              pull={null}
              pulse={false}
              rotation={null}
              size="2x"
              spin={false}
              symbol={false}
              title=""
              transform={null}
            />
          </a>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "aria-label": "linkedin link",
            "children": <FontAwesomeIcon
              border={false}
              className=""
              fixedWidth={false}
              flip={null}
              icon={
                Array [
                  "fab",
                  "linkedin",
                ]
              }
              inverse={false}
              listItem={false}
              mask={null}
              pull={null}
              pulse={false}
              rotation={null}
              size="2x"
              spin={false}
              symbol={false}
              title=""
              transform={null}
            />,
            "href": "https://www.linkedin.com/in/leandrovicentesilva/",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "border": false,
              "className": "",
              "fixedWidth": false,
              "flip": null,
              "icon": Array [
                "fab",
                "linkedin",
              ],
              "inverse": false,
              "listItem": false,
              "mask": null,
              "pull": null,
              "pulse": false,
              "rotation": null,
              "size": "2x",
              "spin": false,
              "symbol": false,
              "title": "",
              "transform": null,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "a",
        },
        "type": "li",
      },
      Object {
        "instance": null,
        "key": "1",
        "nodeType": "host",
        "props": Object {
          "children": <a
            aria-label="github link"
            href="https://github.com/llevvi/"
            rel="noopener noreferrer"
            target="_blank"
          >
            <FontAwesomeIcon
              border={false}
              className=""
              fixedWidth={false}
              flip={null}
              icon={
                Array [
                  "fab",
                  "github",
                ]
              }
              inverse={false}
              listItem={false}
              mask={null}
              pull={null}
              pulse={false}
              rotation={null}
              size="2x"
              spin={false}
              symbol={false}
              title=""
              transform={null}
            />
          </a>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "aria-label": "github link",
            "children": <FontAwesomeIcon
              border={false}
              className=""
              fixedWidth={false}
              flip={null}
              icon={
                Array [
                  "fab",
                  "github",
                ]
              }
              inverse={false}
              listItem={false}
              mask={null}
              pull={null}
              pulse={false}
              rotation={null}
              size="2x"
              spin={false}
              symbol={false}
              title=""
              transform={null}
            />,
            "href": "https://github.com/llevvi/",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "border": false,
              "className": "",
              "fixedWidth": false,
              "flip": null,
              "icon": Array [
                "fab",
                "github",
              ],
              "inverse": false,
              "listItem": false,
              "mask": null,
              "pull": null,
              "pulse": false,
              "rotation": null,
              "size": "2x",
              "spin": false,
              "symbol": false,
              "title": "",
              "transform": null,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "a",
        },
        "type": "li",
      },
      Object {
        "instance": null,
        "key": "2",
        "nodeType": "host",
        "props": Object {
          "children": <a
            aria-label="youtube link"
            href="http://youtube.com/levvitalk"
            rel="noopener noreferrer"
            target="_blank"
          >
            <FontAwesomeIcon
              border={false}
              className=""
              fixedWidth={false}
              flip={null}
              icon={
                Array [
                  "fab",
                  "youtube",
                ]
              }
              inverse={false}
              listItem={false}
              mask={null}
              pull={null}
              pulse={false}
              rotation={null}
              size="2x"
              spin={false}
              symbol={false}
              title=""
              transform={null}
            />
          </a>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "aria-label": "youtube link",
            "children": <FontAwesomeIcon
              border={false}
              className=""
              fixedWidth={false}
              flip={null}
              icon={
                Array [
                  "fab",
                  "youtube",
                ]
              }
              inverse={false}
              listItem={false}
              mask={null}
              pull={null}
              pulse={false}
              rotation={null}
              size="2x"
              spin={false}
              symbol={false}
              title=""
              transform={null}
            />,
            "href": "http://youtube.com/levvitalk",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "border": false,
              "className": "",
              "fixedWidth": false,
              "flip": null,
              "icon": Array [
                "fab",
                "youtube",
              ],
              "inverse": false,
              "listItem": false,
              "mask": null,
              "pull": null,
              "pulse": false,
              "rotation": null,
              "size": "2x",
              "spin": false,
              "symbol": false,
              "title": "",
              "transform": null,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "a",
        },
        "type": "li",
      },
      Object {
        "instance": null,
        "key": "3",
        "nodeType": "host",
        "props": Object {
          "children": <a
            aria-label="instagram link"
            href="http://instagram.com/levvitalk"
            rel="noopener noreferrer"
            target="_blank"
          >
            <FontAwesomeIcon
              border={false}
              className=""
              fixedWidth={false}
              flip={null}
              icon={
                Array [
                  "fab",
                  "instagram",
                ]
              }
              inverse={false}
              listItem={false}
              mask={null}
              pull={null}
              pulse={false}
              rotation={null}
              size="2x"
              spin={false}
              symbol={false}
              title=""
              transform={null}
            />
          </a>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "aria-label": "instagram link",
            "children": <FontAwesomeIcon
              border={false}
              className=""
              fixedWidth={false}
              flip={null}
              icon={
                Array [
                  "fab",
                  "instagram",
                ]
              }
              inverse={false}
              listItem={false}
              mask={null}
              pull={null}
              pulse={false}
              rotation={null}
              size="2x"
              spin={false}
              symbol={false}
              title=""
              transform={null}
            />,
            "href": "http://instagram.com/levvitalk",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "border": false,
              "className": "",
              "fixedWidth": false,
              "flip": null,
              "icon": Array [
                "fab",
                "instagram",
              ],
              "inverse": false,
              "listItem": false,
              "mask": null,
              "pull": null,
              "pulse": false,
              "rotation": null,
              "size": "2x",
              "spin": false,
              "symbol": false,
              "title": "",
              "transform": null,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "a",
        },
        "type": "li",
      },
    ],
    "type": "ul",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <li>
            <a
              aria-label="linkedin link"
              href="https://www.linkedin.com/in/leandrovicentesilva/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <FontAwesomeIcon
                border={false}
                className=""
                fixedWidth={false}
                flip={null}
                icon={
                  Array [
                    "fab",
                    "linkedin",
                  ]
                }
                inverse={false}
                listItem={false}
                mask={null}
                pull={null}
                pulse={false}
                rotation={null}
                size="2x"
                spin={false}
                symbol={false}
                title=""
                transform={null}
              />
            </a>
          </li>,
          <li>
            <a
              aria-label="github link"
              href="https://github.com/llevvi/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <FontAwesomeIcon
                border={false}
                className=""
                fixedWidth={false}
                flip={null}
                icon={
                  Array [
                    "fab",
                    "github",
                  ]
                }
                inverse={false}
                listItem={false}
                mask={null}
                pull={null}
                pulse={false}
                rotation={null}
                size="2x"
                spin={false}
                symbol={false}
                title=""
                transform={null}
              />
            </a>
          </li>,
          <li>
            <a
              aria-label="youtube link"
              href="http://youtube.com/levvitalk"
              rel="noopener noreferrer"
              target="_blank"
            >
              <FontAwesomeIcon
                border={false}
                className=""
                fixedWidth={false}
                flip={null}
                icon={
                  Array [
                    "fab",
                    "youtube",
                  ]
                }
                inverse={false}
                listItem={false}
                mask={null}
                pull={null}
                pulse={false}
                rotation={null}
                size="2x"
                spin={false}
                symbol={false}
                title=""
                transform={null}
              />
            </a>
          </li>,
          <li>
            <a
              aria-label="instagram link"
              href="http://instagram.com/levvitalk"
              rel="noopener noreferrer"
              target="_blank"
            >
              <FontAwesomeIcon
                border={false}
                className=""
                fixedWidth={false}
                flip={null}
                icon={
                  Array [
                    "fab",
                    "instagram",
                  ]
                }
                inverse={false}
                listItem={false}
                mask={null}
                pull={null}
                pulse={false}
                rotation={null}
                size="2x"
                spin={false}
                symbol={false}
                title=""
                transform={null}
              />
            </a>
          </li>,
        ],
        "className": "social-media-list",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "0",
          "nodeType": "host",
          "props": Object {
            "children": <a
              aria-label="linkedin link"
              href="https://www.linkedin.com/in/leandrovicentesilva/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <FontAwesomeIcon
                border={false}
                className=""
                fixedWidth={false}
                flip={null}
                icon={
                  Array [
                    "fab",
                    "linkedin",
                  ]
                }
                inverse={false}
                listItem={false}
                mask={null}
                pull={null}
                pulse={false}
                rotation={null}
                size="2x"
                spin={false}
                symbol={false}
                title=""
                transform={null}
              />
            </a>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "aria-label": "linkedin link",
              "children": <FontAwesomeIcon
                border={false}
                className=""
                fixedWidth={false}
                flip={null}
                icon={
                  Array [
                    "fab",
                    "linkedin",
                  ]
                }
                inverse={false}
                listItem={false}
                mask={null}
                pull={null}
                pulse={false}
                rotation={null}
                size="2x"
                spin={false}
                symbol={false}
                title=""
                transform={null}
              />,
              "href": "https://www.linkedin.com/in/leandrovicentesilva/",
              "rel": "noopener noreferrer",
              "target": "_blank",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "border": false,
                "className": "",
                "fixedWidth": false,
                "flip": null,
                "icon": Array [
                  "fab",
                  "linkedin",
                ],
                "inverse": false,
                "listItem": false,
                "mask": null,
                "pull": null,
                "pulse": false,
                "rotation": null,
                "size": "2x",
                "spin": false,
                "symbol": false,
                "title": "",
                "transform": null,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "a",
          },
          "type": "li",
        },
        Object {
          "instance": null,
          "key": "1",
          "nodeType": "host",
          "props": Object {
            "children": <a
              aria-label="github link"
              href="https://github.com/llevvi/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <FontAwesomeIcon
                border={false}
                className=""
                fixedWidth={false}
                flip={null}
                icon={
                  Array [
                    "fab",
                    "github",
                  ]
                }
                inverse={false}
                listItem={false}
                mask={null}
                pull={null}
                pulse={false}
                rotation={null}
                size="2x"
                spin={false}
                symbol={false}
                title=""
                transform={null}
              />
            </a>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "aria-label": "github link",
              "children": <FontAwesomeIcon
                border={false}
                className=""
                fixedWidth={false}
                flip={null}
                icon={
                  Array [
                    "fab",
                    "github",
                  ]
                }
                inverse={false}
                listItem={false}
                mask={null}
                pull={null}
                pulse={false}
                rotation={null}
                size="2x"
                spin={false}
                symbol={false}
                title=""
                transform={null}
              />,
              "href": "https://github.com/llevvi/",
              "rel": "noopener noreferrer",
              "target": "_blank",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "border": false,
                "className": "",
                "fixedWidth": false,
                "flip": null,
                "icon": Array [
                  "fab",
                  "github",
                ],
                "inverse": false,
                "listItem": false,
                "mask": null,
                "pull": null,
                "pulse": false,
                "rotation": null,
                "size": "2x",
                "spin": false,
                "symbol": false,
                "title": "",
                "transform": null,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "a",
          },
          "type": "li",
        },
        Object {
          "instance": null,
          "key": "2",
          "nodeType": "host",
          "props": Object {
            "children": <a
              aria-label="youtube link"
              href="http://youtube.com/levvitalk"
              rel="noopener noreferrer"
              target="_blank"
            >
              <FontAwesomeIcon
                border={false}
                className=""
                fixedWidth={false}
                flip={null}
                icon={
                  Array [
                    "fab",
                    "youtube",
                  ]
                }
                inverse={false}
                listItem={false}
                mask={null}
                pull={null}
                pulse={false}
                rotation={null}
                size="2x"
                spin={false}
                symbol={false}
                title=""
                transform={null}
              />
            </a>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "aria-label": "youtube link",
              "children": <FontAwesomeIcon
                border={false}
                className=""
                fixedWidth={false}
                flip={null}
                icon={
                  Array [
                    "fab",
                    "youtube",
                  ]
                }
                inverse={false}
                listItem={false}
                mask={null}
                pull={null}
                pulse={false}
                rotation={null}
                size="2x"
                spin={false}
                symbol={false}
                title=""
                transform={null}
              />,
              "href": "http://youtube.com/levvitalk",
              "rel": "noopener noreferrer",
              "target": "_blank",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "border": false,
                "className": "",
                "fixedWidth": false,
                "flip": null,
                "icon": Array [
                  "fab",
                  "youtube",
                ],
                "inverse": false,
                "listItem": false,
                "mask": null,
                "pull": null,
                "pulse": false,
                "rotation": null,
                "size": "2x",
                "spin": false,
                "symbol": false,
                "title": "",
                "transform": null,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "a",
          },
          "type": "li",
        },
        Object {
          "instance": null,
          "key": "3",
          "nodeType": "host",
          "props": Object {
            "children": <a
              aria-label="instagram link"
              href="http://instagram.com/levvitalk"
              rel="noopener noreferrer"
              target="_blank"
            >
              <FontAwesomeIcon
                border={false}
                className=""
                fixedWidth={false}
                flip={null}
                icon={
                  Array [
                    "fab",
                    "instagram",
                  ]
                }
                inverse={false}
                listItem={false}
                mask={null}
                pull={null}
                pulse={false}
                rotation={null}
                size="2x"
                spin={false}
                symbol={false}
                title=""
                transform={null}
              />
            </a>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "aria-label": "instagram link",
              "children": <FontAwesomeIcon
                border={false}
                className=""
                fixedWidth={false}
                flip={null}
                icon={
                  Array [
                    "fab",
                    "instagram",
                  ]
                }
                inverse={false}
                listItem={false}
                mask={null}
                pull={null}
                pulse={false}
                rotation={null}
                size="2x"
                spin={false}
                symbol={false}
                title=""
                transform={null}
              />,
              "href": "http://instagram.com/levvitalk",
              "rel": "noopener noreferrer",
              "target": "_blank",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "border": false,
                "className": "",
                "fixedWidth": false,
                "flip": null,
                "icon": Array [
                  "fab",
                  "instagram",
                ],
                "inverse": false,
                "listItem": false,
                "mask": null,
                "pull": null,
                "pulse": false,
                "rotation": null,
                "size": "2x",
                "spin": false,
                "symbol": false,
                "title": "",
                "transform": null,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "a",
          },
          "type": "li",
        },
      ],
      "type": "ul",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
